cadc-util/src/main/java/ca/nrc/cadc/auth/PosixPrincipal.java

```diff
@@ -43,60 +43,65 @@
  */
 public class PosixPrincipal implements Principal, Serializable
 {
-	private static final long serialVersionUID = 5423257890488724644L;
-	private int uidNumber;
+    private static final long serialVersionUID = 5423257890488724644L;
+    private int uidNumber;
 
     /**
      * Ctor
      * @param uidNumber unique identifier
      */
-    public PosixPrincipal(int uidNumber)
-    {
+    public PosixPrincipal(int uidNumber) {
     	if (uidNumber > 0) {
```

Comment:
Sorry, changed my mind here.  I guess we should allow a uidNumber of zero (but not less than zero) in case someone wants to represent the root user.

And, how about formatting the check like:
```
if (uidNumber <= 0) {
    throw new IllegalArg...
}
this.uidNumber = uidNumber
```
then there's no need for the 'else'